contracts/CO30_Tobias_WhiskeyTangoFoxtrot.vt7/edit_me/briefing/mission.html

```diff
@@ -0,0 +1,3 @@
+The Russian Federation is in Virolahti performing demonstrations with experimental military hardware with the intent to sell.
+Synixe is tasked with collecting intelligence on said hardware, Contractors will be provided vehicles for transportation 
+and a surveillance van capable of listening in on communications and hacking into data points from a distance.
```

Comment:
```suggestion
Synixe is tasked with collecting intelligence on said hardware, Contractors will be provided vehicles for transportation and a surveillance van capable of listening in on communications and hacking into data points from a distance.
```
Must be on a single line or it'll be displayed weird in game